fw/str.c

```diff
@@ -87,14 +87,15 @@ unsigned char custom_ctext_vchar[256] ____cacheline_aligned __read_mostly;
 unsigned char custom_xff[256] ____cacheline_aligned __read_mostly;
 unsigned char custom_cookie[256] ____cacheline_aligned __read_mostly;
 unsigned char custom_etag[256] ____cacheline_aligned __read_mostly;
+unsigned char custom_token_lc[256] ____cacheline_aligned __read_mostly;
 
 #ifdef AVX2
 
 /**
  * Custom alphabets (top and bottom ASCII halves) for AVX2 processing:
  * uri, token, qetoken, nctl, ctext_vchar, xff, cookie, etag.
  */
-unsigned char __CUSTOM[16][32] ____cacheline_aligned __read_mostly = {{0}};
+unsigned char __CUSTOM[17][32] ____cacheline_aligned __read_mostly = {{0}};
```

Comment:
Is 17 correct size? As I see in `__init_custom_a()` we must have two halfs for each alphabet, or am I wrong?